app/controllers/euDetails/DeleteAllEuDetailsController.scala

```diff
@@ -44,13 +44,13 @@ class DeleteAllEuDetailsController @Inject()(
 
   val form: Form[Boolean] = formProvider()
 
-  def onPageLoad(waypoints: Waypoints): Action[AnyContent] = cc.authAndGetData(waypoints.inAmend) {
+  def onPageLoad(waypoints: Waypoints): Action[AnyContent] = cc.authAndGetData(waypoints.registrationModificationMode) {
```

Comment:
```
registrationModificationMode
```
Will equate to amend for everything but the filter checkOtherCountryRegistration where rejoin is treated differently